File: sdks/php/src/Render/Types/GetJobStatusResponseOutput.php
<?php

namespace Pogodoc\Render\Types;

use Pogodoc\Core\SerializableType;
use Pogodoc\Core\JsonProperty;

class GetJobStatusResponseOutput extends SerializableType
{
    /**
     * @var GetJobStatusResponseOutputData $data
     */
    #[JsonProperty('data')]
    public GetJobStatusResponseOutputData $data;

    /**
     * @var GetJobStatusResponseOutputMetadata $metadata
     */
    #[JsonProperty('metadata')]
    public GetJobStatusResponseOutputMetadata $metadata;

    /**
     * @param array{
     *   data: GetJobStatusResponseOutputData,
     *   metadata: GetJobStatusResponseOutputMetadata,
     * } $values
     */
    public function __construct(
        array $values,
    ) {
        $this->data = $values['data'];
        $this->metadata = $values['metadata'];
    }
}

File: sdks/csharp/sdk/src/PogodocApi/Documents/Types/GetJobStatusResponseOutput.cs
using System.Text.Json;
using System.Text.Json.Serialization;
using PogodocApi.Core;

namespace PogodocApi;

public record GetJobStatusResponseOutput
{
    [JsonPropertyName("data")]
    public required GetJobStatusResponseOutputData Data { get; set; }

    [JsonPropertyName("metadata")]
    public required GetJobStatusResponseOutputMetadata Metadata { get; set; }

    /// <summary>
    /// Additional properties received from the response, if any.
    /// </summary>
    /// <remarks>
    /// [EXPERIMENTAL] This API is experimental and may change in future releases.
    /// </remarks>
    [JsonExtensionData]
    public IDictionary<string, JsonElement> AdditionalProperties { get; internal set; } =
        new Dictionary<string, JsonElement>();

    /// <inheritdoc />
    public override string ToString()
    {
        return JsonUtils.Serialize(this);
    }
}
